### PR TITLE
UTC Time used on server, CSRF enabled on forms

### PR DESCRIPTION
**Pull request by kyokidG**

### CODE REVIEW

File: app/game/game_models/player.py [hunk 1]
```diff
@@ -37,17 +37,17 @@ def is_dead(self):
         return False
 
     def is_valid_shot(self):
-        if time.time() - self.last_shot_time > self.shot_cooldown:
+        if (datetime.datetime.utcnow() - self.last_shot_time).total_seconds() > self.shot_cooldown:
             self.total_shots += 1
-            self.last_shot_time = time.time()
+            self.last_shot_time = datetime.datetime.utcnow()
             return True
         else:
```
Comment (cristimihok) [inline app/game/game_models/player.py]: No need for this `else` statement here

File: app/game/game_models/player.py [hunk 2]
```diff
@@ -37,17 +37,17 @@ def is_dead(self):
...
             return False
 
     def is_valid_shield(self):
-        if time.time() - self.last_shield_time > self.shield_cooldown:
+        if (datetime.datetime.utcnow() - self.last_shield_time).total_seconds() > self.shot_cooldown:
             self.shield_activation += 1
-            self.last_shield_time = time.time()
+            self.last_shield_time = datetime.datetime.utcnow()
             return True
         else:
```
Comment (cristimihok) [inline app/game/game_models/player.py]: Redundant `else` statement

File: app/static/css/auth/login.css [hunk 3]
```diff
@@ -50,3 +50,7 @@
     color: rgb(37, 40, 57);
     text-decoration: none;
 }
+
+.title {
+    margin: 10px 0 10px 0;
+}
```
Comment (cristimihok) [inline app/static/css/auth/login.css]: New line

File: app/static/css/auth/register.css [hunk 4]
```diff
@@ -47,3 +47,7 @@
     margin-right: 50%;
     display: block;
 }
+
+.register-title {
+    margin: 10px 0 20px 0;
+}
```
Comment (cristimihok) [inline app/static/css/auth/register.css]: New line

File: app/static/js/game/game_help.js [hunk 5]
```diff
@@ -16,14 +16,18 @@ function startMouseEvents() {
             });
         }
     });
+    game.gameBox.mouseup(function (e) {
+        if (e.which === 3) {
+            socket.emit('230');
+        }
+    });
     $(document).mouseup(function (e) {
         if (e.which === 1) {
```
Comment (cristimihok) [inline app/static/js/game/game_help.js]: Combine this two 'if' statements

File: app/static/js/game/game_help.js [hunk 6]
```diff
@@ -16,14 +16,18 @@ function startMouseEvents() {
...
+            if (currentPlayer.power.css('display') == 'none' ) {
```
Comment (cristimihok) [inline app/static/js/game/game_help.js]: This is wrong. You should't check the state of your application in the UI.

File: app/static/js/game/game_help.js (hunk 5 above)
Comment (kyokidG) [inline app/static/js/game/game_help.js]: Theese are 2 different things. 
If the player raises the left click anywhere inside the document and the powerbar is active, then it shots.
If the player raises the right click JUST IN THE GAME BOX, we activate the shield.

**Pull request merged**